Common/src/main/java/com/techarium/techarium/registry/TechariumItems.java

```diff
@@ -0,0 +1,25 @@
+package com.techarium.techarium.registry;
+
+import com.techarium.techarium.Techarium;
+import com.techarium.techarium.platform.CommonServices;
+import net.minecraft.world.item.BlockItem;
+import net.minecraft.world.item.Item;
+
+import java.util.function.Supplier;
+
+public class TechariumItems {
+
+	public static final Supplier<Item> MACHINE_CORE = CommonServices.REGISTRY.registerItem("machine_core", () -> new BlockItem(TechariumBlocks.MACHINE_CORE.get(), new Item.Properties().tab(Techarium.TAB)));
+	public static final Supplier<Item> COMMUNICATION_DEVICE_ELEMENT = CommonServices.REGISTRY.registerItem("com_device_element", () -> new BlockItem(TechariumBlocks.COMMUNICATION_DEVICE_ELEMENT.get(), new Item.Properties().tab(Techarium.TAB)));
+	public static final Supplier<Item> BOTARIUM = CommonServices.REGISTRY.registerItem("botarium", () -> new BlockItem(TechariumBlocks.BOTARIUM.get(), new Item.Properties().tab(Techarium.TAB)));
+
+	public static final Supplier<Item> EXCHANGE_STATION = Techarium.DEBUG_MODE ? CommonServices.REGISTRY.registerItem("exchange_station", () -> new BlockItem(TechariumBlocks.EXCHANGE_STATION.get(), new Item.Properties().tab(Techarium.TAB))) : () -> null;
```

Comment:
This should not just be registered in debug only just don't add it to a tab if you do not want it to be seen by normal players.